src/ucp/api/ucp.h

```diff
@@ -660,12 +732,11 @@ typedef struct ucp_ep_params {
     uint64_t                field_mask;
 
     /**
-     * Destination address; the address must be obtained using
+     * Destination address; if the UCP_EP_PARAMS_FLAGS_CLIENT_SERVER flag is not
+     * set, this address is mandatory for filling
+     * (along with its corresponding bit in the field_mask -
+     * UCP_EP_PARAM_FIELD_REMOTE_ADDRESS) and must be obtained using
```

Comment:
add @ref to UCP_EP_PARAMS_FLAGS_CLIENT_SERVER and UCP_EP_PARAM_FIELD_REMOTE_ADDRESS